langchain-robomo/app/server.py

```diff
@@ -1,27 +1,26 @@
 import os
 
-from fastapi import FastAPI
+import uvicorn
+from fastapi import BackgroundTasks, FastAPI
 from fastapi.responses import RedirectResponse
 from fastapi.staticfiles import StaticFiles
 from langserve import add_routes
-from .secrets import get_secret_from_env_var_or_secrets_manager
+from rag_momento_vector_index import chain as rag_momento_vector_index_chain
+from rag_momento_vector_index.index import reindex_content
 
+from .secrets import get_secret_from_env_var_or_secrets_manager
 
 os.environ["MOMENTO_API_KEY"] = get_secret_from_env_var_or_secrets_manager(
     secret_env_var_name="MOMENTO_API_KEY",
     secret_name=os.environ.get("MOMENTO_API_KEY_SECRET_NAME"),
     aws_region=os.environ.get("AWS_REGION"),
 )
-print(f"Set MOMENTO_API_KEY to {os.environ['MOMENTO_API_KEY']}")
 os.environ["OPENAI_API_KEY"] = get_secret_from_env_var_or_secrets_manager(
     secret_env_var_name="OPENAI_API_KEY",
     secret_name=os.environ.get("OPENAI_API_KEY_SECRET_NAME"),
     aws_region=os.environ.get("AWS_REGION"),
 )
 
-from rag_momento_vector_index import chain as rag_momento_vector_index_chain
-from rag_momento_vector_index.index import reindex_content
```

Comment:
we unfortunately need this

Reply:
Reverted in [db6f471](https://github.com/momentohq/robo-mo/pull/20/commits/db6f471224877f45817596b47878268553fe07b9)